pkg/kubectl/cmd/util/factory.go

```diff
@@ -57,6 +57,8 @@ type Factory struct {
 	Object func() (meta.RESTMapper, runtime.ObjectTyper)
 	// Returns a client for accessing Kubernetes resources or an error.
 	Client func() (*client.Client, error)
+	// Returns a client for accessing experimental Kubernetes resources or an error.
+	ExperimentalClient func() (*client.ExperimentalClient, error)
```

Comment:
See my other comments. We're not tacking the issue of multiple api groups here. The client api and kubectl both require some rethinking.